web/src/components/Plugin/locales/zh-CN.ts

```diff
@@ -22,6 +22,11 @@ export default {
   'component.plugin.pluginTemplate.tip1': '1. 若路由已配置插件，则插件模板数据将与已配置的插件数据合并。',
   'component.plugin.pluginTemplate.tip2': '2. 插件模板相同的插件会覆盖掉原有的插件。',
 
+  // proxy-mirror
+  'component.pluginForm.proxy-mirror.host.tooltip': '指定镜像服务地址，例如：http://127.0.0.1:9797（地址中需要包含 schema ：http或https，不能包含 URI 部分）',
```

Comment:
When users input an invalid URL, we could popup the error. We don't need this long text actually 😂